linera-core/src/client.rs

```diff
@@ -207,6 +211,51 @@ impl<P, S: Storage + Clone> Client<P, S> {
             },
         }
     }
+
+    pub async fn download_certificates(
```

Comment:
Does this need to be `pub`? (also below)

Reply:
Good catch, it doesn't!